player/lua/stats.lua

```diff
@@ -1358,7 +1358,7 @@ local function update_scale(_, value)
     local scale = 1
     if not o.vidscale then
         if value <= 1 then
-            value = 1
+            value = 720
```

Comment:
`if not o.vidscale and value > 1 then`

Reply:
We can just check `value > 0` since `osd-height` can't be negative or fractional.